patches/openwrt/0009-ath10k-firmware-update-wave-1-firmware-to-10.2.4-1.0-00047.patch

```diff
@@ -0,0 +1,35 @@
+From: David Bauer <mail@david-bauer.net>
+Date: Sat, 5 Oct 2019 12:27:44 +0200
+Subject: ath10k-firmware: update wave 1 firmware to 10.2.4-1.0-00047
+
+This fixes frequent crashes observed on a UniFi AC Mesh using OpenWrt
+master and 19.07. 18.06 seems not affected from our testing.
+
+Signed-off-by: David Bauer <mail@david-bauer.net>
```

Comment:
Why is this patch still here, and not in upstream 19.07?

Reply:
Actually, both of the ath10k firmware patches.

Reply:
I was still a bit unsure about upstreaming this patch, as upstream does not use qca-ath10k anyway. However (as it does not hurt), i will send them upstream too shortly. 

If it does not make it upstream, we have to carry them downstream anyway.

Reply:
maybe that's the only thing keeping us from a merge?

Reply:
I've started a Mailinglist Thread here. As no response came on this, i will submit the patches later today.

http://lists.infradead.org/pipermail/openwrt-devel/2019-October/019380.html

Reply:
https://github.com/openwrt/openwrt/commit/98da3c8cdb6a352a5c74cbbbc948dcc91954fc99
https://github.com/openwrt/openwrt/commit/0e7113e6ece03c879bf99dff097dd868e3453f27